bin/gobblin-mapreduce.sh

```diff
@@ -4,6 +4,9 @@
 ############### Run Gobblin Jobs on Hadoop MR ################
 ##############################################################
 
+# Set during the distribution build
+GOBBLIN_VERSION=@project.version@
```

Comment:
Does this work for you? I added this line, and `echo $GOBBLIN_VERSION` right after, and it just printed out `@project.version@`.

Could it be simper to just include `gobblin-*.jar` in the `libjars`.